src/framework/components/camera/component.js

```diff
@@ -283,12 +286,23 @@ class CameraComponent extends Component {
      */
     requestSceneColorMap(enabled) {
         this._renderSceneColorMap += enabled ? 1 : -1;
+        Debug.assert(this._renderSceneColorMap >= 0);
         const ok = this._enableDepthLayer(enabled);
         if (!ok) {
             Debug.warnOnce('CameraComponent.requestSceneColorMap was called, but the camera does not have a Depth layer, ignoring.');
         }
     }
 
+    set renderSceneColorMap(value) {
+        if (value) {
```

Comment:
one more change. this probably needs to be:
`if (value && !_sceneColorMapRequested) {`
to avoid multiple requests.

Reply:
Done